tfs/src/main/java/hudson/plugins/tfs/model/WorkspaceConfiguration.java

```diff
@@ -132,35 +121,6 @@ else if (cloakedPaths.size() != other.cloakedPaths.size())
             return false;
         else if (!cloakedPaths.containsAll(other.cloakedPaths))
             return false;
-        if (mappedPaths == null) {

```

Comment:
FYI, my guess is the problem with polling might be in here. This is the most complicated "equals" method I have ever seen. We probably need to revisit this entire method.